src/Elcodi/Bundle/CartBundle/Resources/config/doctrine/Order.orm.yml

```diff
@@ -49,6 +49,13 @@ Elcodi\Component\Cart\Entity\Order:
         updatedAt:
             column: updated_at
             type: datetime
+        taxAmount:
```

Comment:
Amount should be defined as integer (see [here](https://github.com/elcodi/elcodi/blob/master/src/Elcodi/Bundle/CartBundle/Resources/config/doctrine/Order.orm.yml#L14)).